.github/workflows/job-postcommit-python-mongodbit.yml

```diff
@@ -0,0 +1,47 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# This workflow will run the integration test of python mongodbio class.
+
+name: Python MongoDBIO IT
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 */6 * * *'
```

Comment:
Is there a reason we added the cron job to these 2 jobs? We don't run these on a schedule currently, right?

Reply:
These jobs run on a scheduled basis in the current configuration

https://ci-beam.apache.org/job/beam_PostCommit_Python_MongoDBIO_IT/
https://ci-beam.apache.org/job/beam_PostCommit_SQL/

Reply:
Oh, you're right I misread this. It looks like they don't run on push though, or am I missing something else? If so, should we remove the `push` trigger?

Reply:
Yes, we can remove the push trigger